scrapers/Hypnotube.yml

```diff
@@ -0,0 +1,28 @@
+name: "Hypnotube"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - hypnotube.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $studio: //a[starts-with(@href,'https://hypnotube.com/user/')]
+    scene:
+      Title: //div[@class='item-tr-inner-col inner-col']/h1/text()
+      Details: //meta[@name='description']/@content
+      Image: //meta[@property='og:image']/@content
```

Comment:
Not working for me (at least with the scenes i checked)
I tried to pull the image from a js that is in the page. It seems to work ok for me.
```yaml
      Image:
        selector: //script[@type="text/javascript" and contains(text(),"twitter:image")]/text()
        postProcess:
          - replace:
              - regex: '.+"twitter:image" content="([^"]+)".+'
                with: $1
```

Reply:
Looks like they changed something on the website. Your method works well. I've added it in my last commit